Content/0.4.6 - Factory Reset/factoryReset.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I understand the 3 levels top also goes for the `X.Y.Z` used in folder names @kugel3 ? Otherwise this could be under `0.4.6.2 - Factory Reset`

Reply:
No the three levels is only for the key, the folder name is not even visible from xcode, it's only purpose is to make it easier to correlate zeplin with the strings, so being exact there is great.